packages/front-end/components/Modal.tsx

```diff
@@ -72,7 +72,7 @@ const Modal: FC<ModalProps> = ({
   fullWidthSubmit = false,
   submitColor = "primary",
   open = true,
-  cta = "Submit",
+  cta = "Save",
```

Comment:
question: This seems like a very broad change -- are we okay with this potentially affecting all Modals?

Reply:
I'll double check with Kari just to be safe, but I recall she mentioned wanting to get rid of the `Submit` language in exchange for `Save`. Thanks!